docs/integrations/sigrid-api-documentation.md

```diff
@@ -910,21 +910,22 @@ Successful response format of this request would look like the following, with t
 }
 ```
 
-### Removing subsystems
+## Removing subsystems
 
 The Sigrid CI script only adds subsystems, they will never disappear by themselves. If you want to remove a subsystem, you can do so via the API.
 
 ```
 POST https://sigrid-says.com/rest/inboundresults/sig/{customer}/{system}/ci/subsystems:batch-delete/v1
 ```
 
-#### Body
 The body of the request is a list of Unix paths that you want to remove from the system.
+
 ```json
     {"names": ["path1, path2"]}
 ```
 
-#### Example request
+The following example shows a complete `curl` commmand that will remove the two sub-systems:
+
 ```bash
 curl --header 'Authorization: Bearer {SIGRID_PERSONAL_TOKEN}' -X POST https://sigrid-says.com/rest/inboundresults/sig/{customer}/{system}/ci/subsystems:batch-delete/v1 -H 'Content-Type: application/json' --data '{ "names" : ["path1", "path2"] }'
```

Comment:
Shouldn't this be: `${SIGRID_PERSONAL_TOKEN}` (note the `$`). With the dollar sign, this is shell variable expansion. Without it, this is either a syntax error, or some kind of meta-syntax to indicate a placeholder.

```suggestion
curl --header 'Authorization: Bearer ${SIGRID_PERSONAL_TOKEN}' -X POST https://sigrid-says.com/rest/inboundresults/sig/{customer}/{system}/ci/subsystems:batch-delete/v1 -H 'Content-Type: application/json' --data '{ "names" : ["path1", "path2"] }'
```